GiEnJul/ClientApp/src/common/components/NavBar.tsx

```diff
@@ -0,0 +1,96 @@
+import {
+  AppBar,
+  Button,
+  IconButton,
+  Toolbar,
+  Drawer,
+  List,
+  ListItem,
+} from "@material-ui/core";
+import React, { FC, useState } from "react";
+import { Link } from "react-router-dom";
+import { Link as Scroll } from "react-scroll";
+import logo from "./../../styling/img/logo_gronn.svg";
+import useStyles from "./Styles";
+import MenuIcon from "@material-ui/icons/Menu";
+import { ArrowForwardIos, Close } from "@material-ui/icons";
+
+const NavBar: FC = () => {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const classes = useStyles();
+
+  const handleEvent = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(() => event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(() => null);
+  };
+  const open = Boolean(anchorEl);
+
+  if (window.location.pathname === "/bli-giver") {
+    return (
+      <>
+        <AppBar className={classes.navContainer}>
+          <Toolbar className={classes.toolBar}>
+            <IconButton >
+              <Link to="/">
+                <img className={classes.smallLogo} src={logo}></img>
+              </Link>
+            </IconButton>
+          </Toolbar>
+        </AppBar>
+      </>
+    );
+  } else {
+    return (
+      <>
+        <AppBar className={classes.navContainer}>
+          <Toolbar className={classes.toolBar}>
+            <IconButton
+              className={classes.navIcon}
+              edge="start"
+              onClick={handleEvent}
+            >
+              <MenuIcon />
+            </IconButton>
+            <Drawer open={open} anchor="top" onClose={handleClose}>
```

Comment:
Nitpick: The `open`probably here could be changed to `open={!!anchorEl}`. I think this would be cleaner, as the code on line 28 could be removed.